src/app/static/src/tests/integration/actions.itest.ts

```diff
@@ -4,32 +4,41 @@ import {DynamicFormMeta} from "@reside-ic/vue-dynamic-form";
 
 describe("actions", () => {
 
-    const getStateWithBaselineOptions = async () => {
+    const getStateWithBaselineSettings = async () => {
```

Comment:
A couple of fixes to the integration test were required for the new behaviour of the prevalence graph data endpoint